ibm/service/power/ibm_pi_constants.go

```diff
@@ -29,7 +29,13 @@ const (
 	Arg_VolumeGroupID                       = "pi_volume_group_id"
 	Arg_VolumeID                            = "pi_volume_id"
 	Arg_VolumeOnboardingID                  = "pi_volume_onboarding_id"
-
+	Arg_VPNPolicyDhGroup                    = "pi_policy_dh_group"
```

Comment:
I would prefer if the name of the variable reflected the value of it.

So "pi_policy_dh_group" would become Arg_PolicyDhGroup.

Reply:
Change made